### DISCUSSION TITLE
是否有必要默认打开tooltip

### DISCUSSION THREAD

**Discussion by xingwanying** [antvis/S2#226] [General]
S2的定位为分析型表格，和折柱饼图属于一个层级，打开tootlip的场景属于多数情况，个人认为应该不需要设置默认打开tooltip,减少用户的配置参数，不用再每次显示设置一次
![image](https://user-images.githubusercontent.com/10885578/132460482-61b8b416-37d7-4dea-a8f5-62c37c53eb2f.png)

**Comment by qubaomingg** [comment]
默认打开 + 1
